common/djangoapps/student/tests/test_refunds.py

```diff
@@ -113,10 +113,10 @@ def test_refundable_with_cutoff_date(self):
         self.assertTrue(self.enrollment.refundable())
 
         with patch('student.models.CourseEnrollment.refund_cutoff_date') as cutoff_date:
-            cutoff_date.return_value = datetime.now() - timedelta(days=1)
+            cutoff_date.return_value = datetime.now(tz=pytz.UTC) - timedelta(minutes=5)
```

Comment:
Nit: you can just pass `pytz.UTC` as a param; `tz` is the only expected argument.